datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -1034,6 +1042,51 @@ private void validateSdxResizeRequest(SdxCluster sdxCluster, String accountId, S
                 });
     }
 
+    private void validateNotInBackupOrRestore(SdxCluster sdxCluster) {
+        if (sdxCluster.getClusterName() != null && ThreadBasedUserCrnProvider.getUserCrn() != null) {
```

Comment:
@RokolyA94 Added an else in there! Thank you! I didn't throw any exceptions because I don't know want to change the overall logic of the whole "resizeSDX" just to do this validation, for example throwing an exception that used to not be there.